src/style/blocks/stars.css

```diff
@@ -0,0 +1,17 @@
+.card__stars {
+  display: flex;
+}
+
+.card__stars-star {
+  background: url(../../images/star.svg) center no-repeat;
+  width: 16px;
+  height: 16px;
+}
+
+.card__stars-star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.card__stars--4 :nth-child(-n+4) {
+  background: url(../../images/star-active.svg) center no-repeat;
+}
```

Comment:
Add possibilty by changing modifire - see differnt stars number on card

Reply:
I've changed it. Is it correct now?